### PR TITLE
Fix KfConfig conversion bugs

### PR DESCRIPTION
**Pull request by gabrielwen**
forgot to copy kind to KfDef in v1beta1.

Related to #3703 

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/4320)

### CODE REVIEW

File: bootstrap/pkg/apis/apps/configconverters/v1beta1.go [hunk 1]
```diff
@@ -12,6 +13,30 @@ import (
...
+		string(kfconfig.AWS_PLUGIN_KIND): true,
+		string(kfconfig.GCP_PLUGIN_KIND): true,
+	}
+
+	_, ok := platforms[pluginKind]
+	return ok
+}
+
+func maybeGetPlatform(pluginKind string) string {
+	platforms := map[string]string{
```
Comment (lluunn) [inline bootstrap/pkg/apis/apps/configconverters/v1beta1.go]: (minor) Should we just use switch?

File: bootstrap/pkg/apis/apps/configconverters/v1beta1.go [hunk 2]
```diff
@@ -12,6 +13,30 @@ import (
 type V1beta1 struct {
 }
 
+func isPlatform(pluginKind string) bool {
```
Comment (lluunn) [inline bootstrap/pkg/apis/apps/configconverters/v1beta1.go]: where is this used?

Comment (gabrielwen) [reply bootstrap/pkg/apis/apps/configconverters/v1beta1.go]: removed.

File: bootstrap/pkg/apis/apps/configconverters/v1beta1.go (hunk 1 above)
Comment (gabrielwen) [reply bootstrap/pkg/apis/apps/configconverters/v1beta1.go]: I don't know, which one is a better solution in golang?

**Pull request merged**